src/css/header.scss

```diff
@@ -1,235 +1,229 @@
-                    &:focus,
-                    &:active,
-                    &.active.focus,
-                    &:active:focus,
-                    &.active,
-                    &.focus {
-                        text-decoration: none;
-
-                        &:before {
-                            display: block;
-                            content: '';
-                            width: 100%;
-                            height: 3px;
-                            position: absolute;
-                            bottom: 0;
-                            left: 0;
-                        }
+            > li {
+                > a {
+                    display: block;
+                    font-weight: 400;
+                    font-size: 13px;
+                    letter-spacing: 0.08em;
+                    color: $body-text-color;
+                    line-height: 20px;
+                    text-transform: uppercase;
+                    cursor: pointer;
+                    transition: color, font-weight 0.25s ease-in;
+                    position: relative;
+
+                    &[href^=http]:after {
+                        content: ' ↗';
                     }
 
-                    &:hover {
-                        &:before {
-                            background-color: $black;
+                    @media (min-width: #{$bp-medium}) {
+                        width: 168px;
+                        padding: 34px 15px;
+
+                        &:hover,
+                        &:focus,
+                        &:active,
+                        &.active.focus,
+                        &:active:focus,
+                        &.active,
+                        &.focus {
+                            text-decoration: none;
+
+                            &:before {
+                                display: block;
+                                content: '';
+                                width: 100%;
+                                height: 3px;
+                                position: absolute;
+                                bottom: 0;
+                                left: 0;
+                            }
                         }
-                    }
 
-                    &:focus,
-                    &:active,
-                    &.active.focus,
-                    &:active:focus,
-                    &.active,
-                    &.focus {
-                        font-weight: 700;
+                        &:hover {
+                            &:before {
+                                background-color: $black;
+                            }
+                        }
 
-                        &:before {
-                            background-color: $link-color;
+                        &:focus,
+                        &:active,
+                        &.active.focus,
+                        &:active:focus,
+                        &.active,
+                        &.focus {
```

Comment:
Goodness gracious.

Realize this is a port-over from what we've already got, but I'll flag that I'm not sure we want necessarily focus and active to be identical. Something to look at more deeply later.

Reply:
Yes, good point!